.github/clang-tidy/test_app/main/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+idf_component_register(SRCS "test_app.c"
+                    INCLUDE_DIRS ".")
```

Comment:
Nitpick: for `idf.py reconfigure` you don't even need this dummy main component. Sure, it will fail in `idf.py build`, but you will probably never do that...

I'm only comparing with https://github.com/espressif/esp-usb/tree/master/clang_tidy ...

Reply:
Have removed, thanks for noticing!